stack.yaml

```diff
@@ -3,6 +3,7 @@ resolver: lts-11.11
 packages:
 - plutus-prototype
 - language-plutus-core
+- plutus-core-interpreter
```

Comment:
also needs to go in default.nix and update the pkgs

Reply:
What should I do with `default.nix`?

Reply:
I'm not sure. `plutus-core-interpreter` is of the same kind as `language-plutus-core`, but the latter only has a line about benchmarks. And I'm getting `[2 built, 0.0 MiB DL]` after I run `nix build -f default.nix plutus-core-interpreter`.

Reply:
Oh, there is another `default.nix`, namely the one in `pkgs`, but I updated it using `pkgs/generate.sh`.

Reply:
You can think of those finding functions as `Package -> Package`. You don't want the benchmark call because you don't have any, hence why I suggested copying `plutus-prototype`.

You did replace the second occurrence of `plutus-prototype` in the kin line, right?

Reply:
You can also run `nix log` with the same arguments to see the build log and check it's doing what you expect.

Reply:
No, I mean the one in the root, the other one is indeed generated.

Reply:
`nix log` gives me a huge wall of text, but I see stuff like:

```
Building library for plutus-core-interpreter-0.1.0.0..
[1 of 1] Compiling Language.PlutusCore.Interpreter.CekMachine ( src/Language/PlutusCore/Interpreter/
ignoring (possibly broken) abi-depends field for packages
Preprocessing test suite 'plutus-core-interpreter-test' for plutus-core-interpreter-0.1.0.0..
Building test suite 'plutus-core-interpreter-test' for plutus-core-interpreter-0.1.0.0..
[1 of 2] Compiling CekMachine       ( test/CekMachine.hs, dist/build/plutus-core-interpreter-test/pl
[2 of 2] Compiling Main             ( test/Main.hs, dist/build/plutus-core-interpreter-test/plutus-c
Linking dist/build/plutus-core-interpreter-test/plutus-core-interpreter-test ...
@nix { "action": "setPhase", "phase": "checkPhase" }
running tests
Running 1 test suites...
Test suite plutus-core-interpreter-test: RUNNING...
Test suite plutus-core-interpreter-test: PASS
```

I didn't touch the `default.nix` file. Also as I said above there is only one line related to `language-plutus-core` and it's about benchmarks, so maybe `default.nix` just shouldn't be touched, because `plutus-core-interpreter` is similar to `language-plutus-core` modulo benchmarks.

Reply:
Here's how it goes:
- Adding it to `pkgs/default.nix` via `generate.sh` puts it in the package set (sorry, I was wrong: I implied that you needed to edit the root `default.nix` before this would happen!)
- The overlay in `/default.nix` tweaks a few properties of our packages.
    - `plutus-prototype` has `addRealTimeLogs` and `filterSource` added.
    - `language-plutus-core` additionally has some stuff relating to benchmarks turned on, and haddocks.
    - We definitely want the first two for `plutus-core-interpreter`, but you probably also want the haddocks.

Here's a diff to apply:
```
diff --git i/default.nix w/default.nix
index 3cf2902..b827893 100644
--- i/default.nix
+++ w/default.nix
@@ -54,6 +54,7 @@ let
       plutus-prototype = addRealTimeTestLogs (filterSource super.plutus-prototype);
       # we want to enable benchmarking, which also means we have criterion in the corresponding env
       language-plutus-core = appendConfigureFlag (doBenchmark (doHaddockHydra (addRealTimeTestLogs (filterSource super.language-plutus-core)))) "--enable-benchmarks";
+      plutus-core-interpreter = doHaddockHydra (addRealTimeTestLogs (filterSource super.plutus-core-interpreter));
     };
   };
   other = rec {
```

Reply:
Thank you, I understand now. Fixed.